guides/GUIDE_FOR_LIBRARY_AUTHORS.md

```diff
@@ -168,6 +168,12 @@ Defaults to `auto`.
 
 Sets the translucency of the status bar (similar to the `StatusBar` component). Defaults to `false`.
 
+#### `swipeDirection` (iOS only)
+
+Sets the direction in which you should swipe to dismiss the screen. The following values are supported:
+- `vertical` – dismiss screen vertically
+- `horizontal` – dismiss screen horizontally (default)
```

Comment:
This could be simplified to a boolean prop as `verticalSwipeEnabled` which by default would be `false`. I doubt there will be any other swipe directions other than vertical and horizontal in the future. Not a strong opinion tho, I'm open to discussing this.

Reply:
I think `verticalSwipeEnabled` might suggest that both options are available when you set it to `true`, while the current implementation strictly lets you choose one of the two.